classes/MyAdminProcess.php

```diff
@@ -64,7 +64,9 @@ public function processExport(&$post, $get)
             if ((isset($post['check']) && count($post['check'])) || Tools::set($post['total-rows'])) {
                 $sql = $where = '';
                 if (Tools::set($post['total-rows'])) { //export whole resultset (regard possible $get limitations)
-                    Tools::dump($get, $this->tableAdmin->fields);exit; //@todo TableLister->generateWhereClause($get)
+                    $columns = $this->tableAdmin->getColumns([]);
+                    $sql = $this->tableAdmin->composeSQL($columns, $_GET);
+                    $sql = $sql['select'];
```

Comment:
Za předchozí syntaxí `Tools::dump($get, $this->tableAdmin->fields);` byl `exit;` a tedy další podmínky v metodě se už nevyhodnocovaly. To už teď opravdu není potřeba?

Reply:
V obou větví if() {...}else{...} se vytvoří $sql, které se dále zpracovává.